src/styles/variables.scss

```diff
@@ -0,0 +1,7 @@
+$black: #060b35;
+$dark-grey: #616070;
+$blue: #00acdc;
+$grey: #f3f3f3;
```

Comment:
Variable names should show their purpose. What if the palette's most used color will be red instead of blue? In this case, you have to find and rename all usage of this color variable

Reply:
Fixed